src/wiki/getting-started/installing-java.md

```diff
@@ -9,24 +9,69 @@ eleventyNavigation:
 
 Java is **required** in order to run Minecraft. As such, until Java downloading is implemented, you **will** need to **manually** install Java to run Minecraft with Prism Launcher.
 
-Currently, we recommend heading over to [Adoptium](https://adoptium.net/) to install the latest versions of Java. Minecraft 1.17 and upwards will require **Temurin 17 (LTS)**, while anything below will require **Temurin 8 (LTS).**
+Oracle Java is not recommended. As of 2019, it has licensing that prohibits certain uses, and it requires an account to download certain versions. Their downloads are intended for those who have purchased a support contract from Oracle.
 
-* 1.17 or newer: **Temurin 17 (LTS)**
-* 1.16.5 or older: **Temurin 8 (LTS)**
+Instead, this guide recommends installing **OpenJDK** from one of the reputable vendors. OpenJDK is a source code only distribution of Java made by a collaboration between Oracle and industry partners such as Adoptium, Microsoft, Red Hat, and Azul. Unlike Oracle Java, it is released under a license that minimizes legal risk for users and does not require payment.
 
-If you aren't sure which version you will need, **it is recommended to just install both.**
+If you would like to learn more about OpenJDK and the vendors mentioned in this page, here are their websites:
 
-**NOTE:** You can use any OpenJDK build you want. [Azul Zulu](https://www.azul.com/downloads/?package=jre#download-openjdk) is a popular alternative to Temurin, and will work just as well.
+- [Adoptium Temurin](https://adoptium.net/)
+- [Microsoft OpenJDK](https://www.microsoft.com/openjdk)
+- [Azul OpenJDK](https://www.azul.com/downloads/)
+- [OpenJDK source distribution](https://openjdk.org/)
 
-You can also use Oracle Java, but openjdk is better here. Also, to download Oracle Java 9+ you need an Oracle account which is not a good thing.
+Otherwise, continue reading for quick download links and a guide for which one to choose.
 
-## Java on linux
+## Installing Java on Windows
+
+**First, check your CPU architecture.** Here is a method that works on all Windows versions:
+
+1. Open command prompt
```

Comment:
```suggestion
1. Open command prompt by pressing Windows + R and then entering cmd
```
This would help new users who might not know what command prompt is

Reply:
Pressing Win + R does not work on touchscreen-only Windows devices, only the Terminal app. So we probably need general instruction to go to the start menu and search for "Command Prompt" or "Terminal".

Reply:
Good idea

Reply:
Done in 0c8ce0ce15cf080b823014a1a3e8f7e7f056bfaa